src/codeu/chat/util/Duration.java

```diff
@@ -0,0 +1,28 @@
+package codeu.chat.util;
+
+public class Duration implements Comparable<Duration> {
+  public final double offset;
+  public final int scale;
```

Comment:
I'd prefer to use a single `long` to store duration in ms, to make it compatible with `Time`. You also don't have to deal with infinities/denormalized doubles in compareTo.